website/integrations/services/actual-budget/index.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Integrate with Actual Budget
+sidebar_label: Actual Budget
+---
+
+# Actual Budget
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Actual Budget
+
+> Actual Budget is a super-fast and privacy-focused app for managing your finances. At its core is the well-proven and much-loved envelope budgeting methodology.
+> You own your data and can do whatever you want with it. It features multi-device sync, optional end-to-end encryption, and much more.
+>
```

Comment:
Normally I would rewrite the markety-hype description of the product, and state plainly what the product does, common use cases, etc. However, since this is a direct quote from the product's website, we can leave it for now. As a future project, we will probably rewrite all of them.

Reply:
I can do it for this PR as well, we don't have to wait 😂